app.py

```diff
@@ -107,7 +117,8 @@ def getData():
 		feature["properties"]["price"] = record.price
 		feature["geometry"]["coordinates"] = [record.latitude, record.longitude]
 
-		output["features"].append(feature)
+        feature["properties"]["normprice"] = remap(record.price, minprice, maxprice, 0, 1)
+        output["features"].append(feature)
```

Comment:
You need to make sure that these two lines are inset one more level so that they are in the loop and are run for every record.